files/.github/workflows/stale.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         stale-issue-message: 'Oops, seems like we needed more information for this issue, please comment with more details or this issue will be closed in 7 days.'
         close-issue-message: 'This issue was closed because it is missing author input.'
         stale-issue-label: 'kind/stale'
-        any-of-labels: 'hint/needs-author-input,need/author-input'
+        any-of-labels: 'need/author-input'
         days-before-issue-stale: 6
         days-before-issue-close: 7
```

Comment:
:information_source: 
With this, we can mark issues as `need/author-input` during the triage, and have automatic cleanup.
If there is no response within 6 days, bot will ping the author + mark the issue as `kind/stale`. 
This allows us to filter-out such issues during triages, skipping issues that had no updates.

Reply:
If I understood the logic correctly, are we only giving the author one day to add more information to the issue? If that is the case, I would raise the number of days to close the issue, giving more time to the users.

Reply:
Nope, it's 6 days until the issue is marked as stale and then, if no action happens on the issue, it is closed after additional 7 days. So it's 6 + 7.

Reply:
Thanks for the explanation.